hacker/index.html

```diff
@@ -6,6 +6,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Hacker | Free Professional Portfolio Website Templates by Jofferson Tiquez</title>
+  <link rel="shortcut icon" href="./hacker.png" type="image/x-png">
```

Comment:
Kindly use favicon generator and apply the proper meta for all screen sizes and types of web app. Mobile, tablet, desktop. PWA, etc.

https://www.favicon-generator.org/

Reply:
> Kindly use favicon generator and apply the proper meta for all screen sizes and types of web app. Mobile, tablet, desktop. PWA, etc.
> 
> https://www.favicon-generator.org/

Done Fixed.